multicluster/controllers/multicluster/memberclusterannounce_controller.go

```diff
@@ -163,6 +142,7 @@ func (r *MemberClusterAnnounceReconciler) Reconcile(ctx context.Context, req ctr
 		klog.ErrorS(err, "Failed to add member cluster to ClusterSet", "cluster", memberAnnounce.ClusterID)
 		return ctrl.Result{}, err
 	}
+	r.addMemberStatus(memberID, v1.ConditionTrue)
 	klog.InfoS("Adding finalizer to MemberClusterAnnounce", "MemberClusterAnnounce", klog.KObj(memberAnnounce))
```

Comment:
We will repeatedly log this message for every memberAnnounce update. Should should check if Finalizer is already added?

Reply:
When the finalizer exists, it will return, code won't reach here.
```
	if common.StringExistsInSlice(memberAnnounce.Finalizers, finalizer) {
		return ctrl.Result{}, nil
	}
```